.github/workflows/scan-pull-request.yml

```diff
@@ -0,0 +1,47 @@
+name: "Frogbot Scan Pull Request"
+
+on:
+  workflow_call:
+    inputs:
+      upstream-packages:
+        description: A whitespace separated list of Howso package dependencies (e.g., "amalgam-lang-py howso-engine-py").
+        type: string
+        required: false
+
+permissions:
+  pull-requests: write
```

Comment:
Does this permission mean it will update the PR with the actual errors it finds?

Reply:
Ah, this was a leftover from when I tried using the Frogbot action (which didn't work for a few reasons). That one _did_ write comments on PRs when something failed.